L10-OptionalLocalDateRegex/L10-Zad.3-URLValidator/src/main/java/pl/babiarz/URLValidator.java

```diff
@@ -0,0 +1,32 @@
+package pl.babiarz;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class URLValidator {
+
+    public static boolean isUrlValid(String url){
+        Pattern pattern = Pattern.compile("((http|https)://)(www.)?"
+                + "[a-zA-Z0-9@:%._\\+~#?&//=]"
+                + "{2,256}\\.[a-z]"
+                + "{2,6}\\b([-a-zA-Z0-9@:%"
+                + "._\\+~#?&//=]*)");
+        Matcher matcher = pattern.matcher(url);
+
+        return matcher.matches();
+    }
+
+    public static void main(String[] args) {
+        String url1 = "http://www.onet.pl";
+        String url2 = "https://mail.google.com";
+        String url3 = "http://wiadmosci.onet.pl";
+        String url4 = "http://onet.pl";
+
+
+        System.out.println(URLValidator.isUrlValid(url1));
```

Comment:
w sumie patrząc na wywołanie to wystarczyłoby, aby nazwa metody była 'isValid' zamiast 'isUrlValid', bo i tak widzimy:

'URLValidator.isValid' więc wiadomo co walidujemy;)